src/uct/ib/ud/base/ud_iface.h

```diff
@@ -1,5 +1,7 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
```

Comment:
pls remove

Reply:
sure. useful for emacs to keep the file consistent with the intended style but I don't see any reason I will be editing these files again any time soon. removing in next push.